Cargo.toml

```diff
@@ -227,3 +227,13 @@ pprof = "0.13"
 proptest = "1.4"
 proptest-derive = "0.4"
 serial_test = "2"
+
+# Linting rules across all crates in the workspace.
+[workspace.lints.rust]
+missing_debug_implementations = "warn"
+missing_docs = "warn"
+unreachable_pub = "warn"
+"rustdoc.all" = "warn"
```

Comment:
Is it working? Shouldn't it be something like:

```
[workspace.lints]
rust.missing_debug_implementations = "warn"
rust.missing_docs = "warn"
rust.unreachable_pub = "warn"
rustdoc.all = "warn"
rust.unused_must_use = "deny"
rust.rust_2018_idioms = "deny"
```

Reply:
this is fine except for rustdoc which is in lints.rustdoc not lints.rust.rustdoc

Reply:
Reverted to wotk without the name prefixes. 

> Is it working? Shouldn't it be something like:
> 
> ```
> [workspace.lints]
> rust.missing_debug_implementations = "warn"
> rust.missing_docs = "warn"
> rust.unreachable_pub = "warn"
> rustdoc.all = "warn"
> rust.unused_must_use = "deny"
> rust.rust_2018_idioms = "deny"
> ```

Yes, it's just a name prefix but I removed the changes, as it's a bit light on [documentation](https://rust-lang.github.io/rfcs/3389-manifest-lint.html) around how to use it.